tests/foreman/ui/test_subscription.py

```diff
@@ -554,3 +556,77 @@ def test_positive_prepare_for_sca_only_subscription(target_sat, function_entitle
             'This organization is not using Simple Content Access. Entitlement-based subscription '
             'management is deprecated and will be removed in Satellite 6.16.' in sca_alert
         )
+
+
+@pytest.mark.parametrize('setting_update', ['expire_soon_days'], indirect=True)
+def test_positive_check_manifest_validity_notification(
+    target_sat, setting_update, default_org, function_org
+):
+    """Check notification when manifest is going to expire.
+
+    :id: 29ab0e80-25eb-44f0-9294-3a26922c33c5
+
+    :setup:
+        1. create new organization, let's say "Expire manifest org"
+
+    :steps:
+        1. Upload expired manifest in newly created org
+        2. go to Content > Subscriptions page, click on 'Manage Manifest' button.
+        3. Search for message string 'Manifest expired', or 'Your manifest expired'
+        4. delete expired manifest from this org (cleanup part)
+        5. Upload non-expired manifest in default org
+        6. go to Content > Subscription page, click on 'Manage Manifest' button.
+        7. Search for message string 'Manifest expiring soon', or 'Your manifest will expire'
+        8. delete expired manifest from default org (cleanup part)
+
+    :expectedresults:
+        1. 'Manifest expired' message appear on screen
+
+    :BZ: 2075163
+
+    :customerscenario: true
+    """
+    property_name = setting_update.name
+    property_value = 366  # value should be greate than 365 to get expected output message
+    remote_path = f'/tmp/{EXPIRED_MANIFEST}'
+    target_sat.put(DataFile.EXPIRED_MANIFEST_FILE, remote_path)
+    # upload expired manifest
+    target_sat.cli.Subscription.upload({'organization-id': function_org.id, 'file': remote_path})
+
+    # Create new manifest
+    _, temporary_local_manifest_path = mkstemp(prefix='manifest-', suffix='.zip')
+    with clone() as manifest, open(temporary_local_manifest_path, 'wb') as file_handler:
+        file_handler.write(manifest.content.read())
+
+    with target_sat.ui_session() as session:
```

Comment:
Is there a specific reason we are using `satellite.ui_session`, 
instead of importing `session` fixture?

Reply:
Importing session creates new org, new loc, which is not needed.
To get Manifest expired message we need org with no manifest imported. So prefer to use 'ui_session'